packages/shared/routes/dashboard/settings/views/Security.svelte

```diff
@@ -344,7 +349,7 @@
             <Text type="p" secondary smaller classes="mb-2">{locale('views.settings.changePincode.newPincode')}</Text>
             <Pin smaller error={newPincodeError} classes="mb-4" bind:value={newPincode} disabled={pinCodeBusy} />
             <Text type="p" secondary smaller classes="mb-2">{locale('views.settings.changePincode.confirmNewPincode')}</Text>
-            <Pin smaller error={confirmationPincodeError} classes="mb-4" bind:value={confirmedPincode} disabled={pinCodeBusy} />
+            <Pin smaller error={confirmationPincodeError} classes="mb-4" bind:value={confirmedPincode} disabled={pinCodeBusy} on:submit={changePincode} />
```

Comment:
Wouldnt you add the onSubmit to all, pin and password, so if you edit something you can submit from whichever field you are in?

Reply:
I guess we could, under normal conditions it won't allow a submit unless the others are complete, but on correcting something that could be the case, I will update.